src/NUnitEngine/nunit.engine/Services/TestAgency.cs

```diff
@@ -195,6 +296,9 @@ public void StartService()
                 _runtimeService = ServiceContext.GetService<IRuntimeFrameworkService>();
                 _availableRuntimeService = ServiceContext.GetService<IAvailableRuntimes>();
 
+                _launchers.Add(new Net462AgentLauncher());
```

Comment:
Do we need a Mono runner? For .NET462 support on Linux

Reply:
We do. But that would be a function of the .NET Framework runner. Suport is currently commented out. I have a TODO for adding the support back.